### PR TITLE
Issue 449 newdocs to reflect the new security evaluation process from issue 449

### PR DESCRIPTION
**Pull request by magnologan**
LGTM.

Comments and suggestions added to the thread in Slack:

https://cloud-native.slack.com/archives/C01AZ5RDD8F/p1604508284042800

### CODE REVIEW

**Comment by TheFoxAtWork** [comment]
@magnologan i think i have all the new docs added to this.  take a look?

File: assessments/guide/self-assessment.md
Comment (lumjjb) [inline assessments/guide/self-assessment.md:45]: would prefer to use reserved names for domains so that there's no potential future domain name hijacking possible. Maybe example.org/file or example.org/config

File: assessments/guide/joint-readme-template.md
Comment (lumjjb) [inline assessments/guide/joint-readme-template.md:1]: I think it would be nice to have some examples of some of these fields, or a guide on what type of content goes into the fields. Maybe links to previous evaluations, etc. Could be part of this document or part of the guide doc. I think you've done a great job in the joint evaluation in doing that.

File: assessments/guide/joint-evaluation.md
Comment (lumjjb) [inline assessments/guide/joint-evaluation.md:163]: would be good to also list completed security audit and audit reports.

File: assessments/guide/joint-evaluation.md
Comment (lumjjb) [inline assessments/guide/joint-evaluation.md:167]: I'm not sure if these standards are what we should be asking about. These mostly relate to products or deployments, which may not be applicable to most of the evaluations we do. I'm thinking that based on what we are trying to find out here: Can I use this for compliance, or what is the current evaluated security posture of the project?

For the first, we can ask for compliance mapping mainly targeted at NIST 800-53, or HiTrust. And for the later, examples of use of projects in compliant environments or security audits.

**Review by lumjjb — APPROVED** [review]
Great work! Have a couple comments for discussion, LGTM

**Comment by TheFoxAtWork** [comment]
@lumjjb think i got this all updated for you please take a look.

**Comment by TheFoxAtWork** [comment]
change evaluation to review....

**Comment by lumjjb** [comment]
>change evaluation to review....

? what do you mean by this, proposing security evaluation -> security review?

**Comment by ultrasaurus** [comment]
We were chatting and I commented that "evaluation" seemed (to me) to apply that we would evaluate whether an open source project was secure, which is not really what we're doing -- we review it for security concerns, identify issues / boundaries, etc.  So then @TheFoxAtWork thought we could call it a "Security Review," which I felt addressed the issue I had with other naming

**Comment by lumjjb** [comment]
Gotcha - yea security review is fine. Thanks for providing the context

**Comment by TheFoxAtWork** [comment]
sorry yea this is on my TODO list.  i'm swamped today though and next week looks like a nightmare waiting to attack.

**Comment by lumjjb** [comment]
tag #449 for ref

**Review by lumjjb — APPROVED** [review]
LGTM

**Comment by TheFoxAtWork** [comment]
this is now OBE, merged into #488 for a clean update.

**Pull request closed without merging**
